skrub/tests/test_similarity_encoder.py

```diff
@@ -282,7 +208,7 @@ def test_fit_transform() -> None:
 
 def test_get_features() -> None:
     # See https://github.com/dirty-cat/skrub/issues/168
```

Comment:
The link to the issue above seems wrong (I know it's independent of your PR, but let's fix this)